waspc/examples/todoApp/ext/serverSetup.js

```diff
@@ -10,10 +10,10 @@ const setup = async () => {
   console.log('Custom server setup done!')
 
   console.log('Kicking off Job...')
-  // Or: const runningJob = mySpecialJob.delay(1000).performAsync({ something: "here" })
-  const runningJob = mySpecialJob.performAsync({ something: "here" })
-  console.log('Waiting for Job result...')
-  runningJob.result.then(res => { console.log(res) }).finally(() => { console.log("Job done!") })
+  // Or: const scheduledJob = await mySpecialJob.delay(10).submit({ something: "here" })
+  const scheduledJob = await mySpecialJob.submit({ something: "here" })
+  console.log("Job:", scheduledJob)
+  console.log("scheduledJob.pgBoss.details()", await scheduledJob.pgBoss.details())
```

Comment:
Would you expect to see `scheduledJob.pgBoss.details()` or `scheduledJob.PgBoss.details()`, given that the executor is uppercase `PgBoss`?